packages/SystemsThinkingWorkbench-Core.package/STWPlayfield.class/instance/addSimulationButtons.st

```diff
@@ -0,0 +1,15 @@
+initialization
+addSimulationButtons
+	
+	self playButton: STWButton new.
+	self playButton contents: 'Start'.
+	self playButton clickHandler: [ self toggleSimulation. ].
+	self addMorph: self playButton.
+	self playButton position: self position.
```

Comment:
... FAULPELZ i lay my both hands in the grill, that he wanted them there but okay